measure/endurance-snapshot

```diff
@@ -475,8 +467,22 @@ fi
 echo "- Disk usage"
 LC_ALL=C df -k -P > $path/df
 
-echo "- statefs"
-dump_statefs_value namespaces/Screen/Blanked
+echo "- mce"
```

Comment:
How about:

```
if type mcetool >/dev/null 2>/dev/null ; then
    echo "- mce"
    ...
fi

```

Reply:
Thanks, yes, I've made this change and pushed a new version (and also fixed some other things that I accidentally messed up).